app/controllers/public/index.js

```diff
@@ -6,10 +6,14 @@ export default Controller.extend({
 
   speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
Why two arrays ? The API response is going to be a single array of speakers. Use the `isFeatured` flag to differentiate.